Kconfig

```diff
@@ -99,6 +99,37 @@ config KAS_INCLUDE_LXDE
 
 endif
 
+# Provide two entries so that the help text can be adjusted to the image.
+config SECURE_BOOT
+	bool "Secure boot"
+	depends on IMAGE_SWUPDATE
+	help
+	  Enable signing of boot artifacts (boot loader, unified kernel image).
+	  Furthermore activate dm-verity integrity protection for the read-only
+	  root filesystem.
+
+	  This uses the public custMpk.key and certificate by default. Do not
+	  use this key in production, it is for demonstration purposes only.
+
+	  Note: The persistent /var and /home partitions are not yet protected!
+
+config SECURE_BOOT
+	bool "Secure boot"
+	depends on IMAGE_BOOT_PG1 || IMAGE_BOOT_PG2
```

Comment:
PG1 will not support secure boot. Should we need a disclaimer to record this information somewhere?